docs/source/notebooks/DataTransformers.md

```diff
@@ -543,3 +549,297 @@ clf.fit(x_train, y_train)
 score = clf.score(x_test, y_test)
 print(f"Test score (accuracy): {score:.2f}")
 ```
+
+## 6. Ragged datasets with tf.data.Dataset
+
+SciKeras provides a third dependency injection point that operates on the entire dataset: X, y & sample_weight. This `dataset_transformer` is applied after `target_transformer` and `feature_transformer`. One use case for this dependency injection point is to transform data from tabular/array-like to the `tf.data.Dataset` format, which only requires iteration. We can use this to create a `tf.data.Dataset` of ragged tensors.
+
+Note that `dataset_transformer` should accept a single **3 element tuple** as its argument and return value:
+
+```python
+help(KerasClassifier.dataset_transformer)
```

Comment:
Style nit: link to the docs on this.

Reply:
Instead of calling `help(...)` or in addition to?

Reply:
I'd definitely link to the docs. 

I'm not sure calling `help` does anything meaningful. It outputs the docstring, right? I'm not sure that's super meaningful, unless you refer to a specific point later in this notebook.

Reply:
I think just linking to the docstring within the docs (like [this](https://www.adriangb.com/scikeras/refs/heads/master/generated/scikeras.wrappers.KerasClassifier.html#scikeras.wrappers.KerasClassifier.feature_encoder)) should suffice. The information on the signature and such is important to understand the notebook, but I wanted to avoid duplicating the text. I'll rewrite it as a 1-2 sentence explanation in the notebook + a link to the docs.